vosekast_control/Scale.py

```diff
@@ -178,13 +186,15 @@ def _scale_input_buffer(self):
                 sleep(0.1)
 
         elif self.emulate:
-            self.scale_input_buffer.appendleft(b'+ 0.000 kg')
+            self.scale_input_buffer.appendleft(0)
             while self.is_running:
                 scale_input = 0.0 + uniform(0.0, 0.2)
                 self.scale_publish = True
                 self.scale_input_buffer.appendleft(scale_input)
                 sleep(1)
 
+                sleep(0.05)
```

Comment:
this has already been fixed, see line 194

Reply:
yeah right, that is the product of an merge, i will remove that